src/avfoundation.cs

```diff
@@ -11274,6 +11533,14 @@ interface AVSampleBufferDisplayLayer {
 		[Field ("AVSampleBufferDisplayLayerFailedToDecodeNotificationErrorKey")]
 		NSString FailedToDecodeNotificationErrorKey { get; }
 
+		[TV (11, 0), NoWatch, Mac (10, 13), iOS (11, 0)]
+		[Field ("AVSampleBufferAudioRendererWasFlushedAutomaticallyNotification")]
+		NSString AudioRendererWasFlushedAutomaticallyNotification { get; }
```

Comment:
the naming makes it sound like a `[Notification]` attribute is missing (and intro complains about this)